index.html

```diff
@@ -3053,15 +3053,105 @@ <h2>Identifier</h2>
+            The following constraints MUST be
+            applied to the event payload:
+          </span>
+        </p>
+        <table class="def">
+          <thead>
+            <tr>
+              <th>keyword</th>
+              <th>type</th>
+              <th>type constraints</th>
+              <th>value constraints</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>specversion</td>
+              <td>string</td>
+              <td>mandatory, non-empty</td>
+              <td>The <code>specversion</code> MUST have the value "1.0"</td>
+            </tr>
+            <tr>
+              <td>id</td>
+              <td>string</td>
+              <td>mandatory, non-empty</td>
+              <td>For each distinct event the combination of <code>source</code> + <code>id</code> MUST be unique. 
+                A WebThing MAY resend the same event with the same <code>id</code>. 
+                If an event is resent with the same <code>id</code>, a consumer SHOULD treat it as
+                duplicate.</td>
+            </tr>
+            <tr>
+              <td>source</td>
+              <td>uri</td>
+              <td>mandatory, non-empty</td>
+              <td>The combination of <code>source</code> + <code>id</code> MUST be unique for each distinct event. 
+                The source MUST be set to the <code>baseURI</code> of the WebThing.</td>
+            </tr>
+            <tr>
+              <td>subject</td>
+              <td>string</td>
+              <td>optional</td>
+              <td>When this field is used, it MUST be set to the <code>title</code> of the event or property
+                affordance.</td>
+            </tr>
+            <tr>
+              <td>type</td>
+              <td>string</td>
+              <td>mandatory, non-empty</td>
+              <td>For event affordances the source MUST be set to the string "/events/<code>eventName</code>" of the
+                event affordance.</br>
+                For property affordances the source MUST be set to the string "/properties/<code>propertyName</code>" of
+                the property affordance.
+              </td>
+            </tr>
+            <tr>
+              <td>time</td>
+              <td>string</td>
+              <td>optional, compliant with [[RFC3339]]</td>
+              <td>time when the event occurred. If the WebThing does not have a clock, it MUST use the same
+                algorithm on all event affordances to determine the value of this field.
+            </tr>
+            <tr>
+              <td>datacontenttype</td>
+              <td>string</td>
+              <td>optional, [[RFC2046]] compliant MIME type, when set the value MUST be "<code>application/json</code>"</td>
+              <td>If a WebThing contains a <code>data member</code>, this field MUST be present.</td>
+            </tr>
+            <tr>
+              <td>data</td>
+              <td>string</td>
```

Comment:
why do we have a `string` here?

Reply:
![image](https://user-images.githubusercontent.com/13832739/181181219-4799ee8d-ec6e-4758-9a45-211daccd7afa.png)

I think, it should be definitely any type.